package.json

```diff
@@ -34,8 +34,10 @@
 		"fix": "npm run format:fix && npm run lint:fix"
 	},
 	"devDependencies": {
+		"@jazzer.js/hooking": "^1.1.0",
```

Comment:
Same as with `core`, where is this dependency needed?

Reply:
done